appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/DesignToolbar.ui.xml

```diff
@@ -27,13 +28,16 @@
       <ya:AddFormAction />
     </ai:ToolbarItem>
     <ai:ToolbarItem name="RemoveForm" caption="{messages.removeFormButton}" ui:field="removeFormItem"
-                    align="center">
+       align="center">
       <ya:RemoveFormAction />
     </ai:ToolbarItem>
-    <ai:ToolbarItem name="ProjectPropertiesDialog" caption="{messages.projectPropertiesText}" ui:field="projectPropertiesDialog"
-                    align="center">
+    <ai:ToolbarItem name="ProjectPropertiesDialog" caption="{messages.projectPropertiesText}" ui:field="projectPropertiesDialog" align="center">
```

Comment:
Not sure why this was changed since it has nothing to do with the functionality addressed in this PR. It also makes the line longer than the regular line length limit.